app/src/main/java/org/schabi/newpipe/player/BasePlayer.java

```diff
@@ -327,6 +331,8 @@ protected void initPlayback(@NonNull final PlayQueue queue,
 
         if (playQueueAdapter != null) playQueueAdapter.dispose();
         playQueueAdapter = new PlayQueueAdapter(context, playQueue);
+
+        if (isMuted) simpleExoPlayer.setVolume(0);
```

Comment:
Just to make sure the volume is at maximum, since you removed this line: https://github.com/TeamNewPipe/NewPipe/pull/3165/files#diff-0b87dfa694cf6a7720aee912b16ab32dL392
```suggestion
        simpleExoPlayer.setVolume(isMuted ? 0 : 1);
```